src/main/app/src/app/shared/pipes/vertrouwelijkaanduiding-to-translation-key.pipe.ts

```diff
@@ -0,0 +1,37 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { Vertrouwelijkheidaanduiding } from "../../informatie-objecten/model/vertrouwelijkheidaanduiding.enum";
+
+@Pipe({
+  name: "vertrouwelijkaanduidingToTranslationKey",
+  standalone: true,
+})
+export class VertrouwelijkaanduidingToTranslationKeyPipe
+  implements PipeTransform
+{
+  public expectedKeys = [
+    "openbaar",
+    "beperkt_openbaar",
+    "intern",
+    "zaakvertrouwelijk",
+    "vertrouwelijk",
+    "confidentieel",
+    "geheim",
+    "zeer_geheim",
+  ] as const;
+
+  transform(
+    value: string | (typeof this.expectedKeys)[0],
+  ): `vertrouwelijkheidaanduiding.${Vertrouwelijkheidaanduiding}` {
+    if (!this.expectedKeys.includes(value as any)) {
+      throw new Error(`Unexpected vertrouwelijkheidaanduiding: ${value}`);
```

Comment:
We should think about the error messages, this is Dutch and English mixed. Maybe in a json format? Then the UI code can transform it again in a "nice" message...

Reply:
This will only be logged to the console, the user should not be able to reproduce this